packages/plugin-board/src/board.js

```diff
@@ -127,6 +130,28 @@ const Board = SparkPlugin.extend({
     }, channel);
   },
 
+  _ensureOnlyConversationAuthorization(channel) {
+    if (!this._isChannelCreatorAuthorized(channel.creatorId, channel.kmsMessage)) {
+      return Promise.resolve(channel);
+    }
+
+    // remove channel creator from board service kms as we want to link to
+    // conversation KRO only
+    const kmsMessage = {
```

Comment:
Good catch, Ian! The code is super nice after combining `process.nextTick` and `tap` 👍